microhapulator/workflows/preproc-paired.smk

```diff
@@ -80,7 +80,6 @@ rule merge:
         r2=rules.filter_ambiguous.output.filtered_r2,
     output:
         mergedfq="analysis/{sample}/{sample}.extendedFrags.fastq",
-        linkedfq="analysis/{sample}/{sample}-preprocessed-reads.fastq",
```

Comment:
`{sample}-preprocessed-reads.fastq` is now linked after length filtering. This file is the input for the rest of the workflow.